src/utils/escape.ts

```diff
@@ -34,6 +34,11 @@ export function escapeProperty(propName: string): string {
     return escapeIfNeeded(propName);
 }
 
+/** Escapes a variable name if needed */
+export function escapeVariable(varName: string): string {
+    return escapeIfNeeded(varName);
+}
```

Comment:
Do we ever envision a world where escaping for types, labels, properties and variables have different behaviours?

Reply:
Well, for a while it was different, hence these separate methods. I can imagine some edge cases we may want to treat differently and I think it makes sense, in terms of separation of concepts, to have separate functions, even if all of those pipe to the same function atm